### PR TITLE
jakarta support

### PR DESCRIPTION
**Pull request by p-8-z**
Jakarta support for easy-random-bean-validation

### CODE REVIEW

**Comment by fmbenhassine** [comment]
Yes! I am planning to merge this PR and release a new version this June. Stay tuned!

**Comment by p-8-z** [comment]
@fmbenhassine it's almost end of the June, is it still in plan?

**Comment by fmbenhassine** [comment]
Yes! Thank you for the PR, rebased and merged as c4d2a3c0b8c6edb80e9ec6f832b75268c42f535f. I just changed the `jakarta.el` version from 5.0.0-M1 to 4.0.2 as we can't release with a milestone dependency.

v6 snapshots are [available](https://oss.sonatype.org/content/repositories/snapshots/org/jeasy/easy-random-core/6.0.0-SNAPSHOT/) with this change included, I would be grateful if you give them a try and tell me if you have any issue.

The main feature I am planning to include besides this one is #397. After that, v6 should be ready to release (and which will be based on Java 17, #495).

File: easy-random-core/src/test/java/org/jeasy/random/EasyRandomTest.java [hunk 1]
```diff
@@ -231,6 +231,7 @@ void nextEnumShouldNotAlwaysReturnTheSameValue() {
     }
 
     @Test
+    // this test fails on jdk17
```
Comment (fmbenhassine) [inline easy-random-core/src/test/java/org/jeasy/random/EasyRandomTest.java:234]: Thank you for raising this, I also noticed that. I created #494 to investigate the issue.

**Comment by fmbenhassine** [comment]
After careful consideration, I decided to discontinue the bean validation module, see #499.

As mentioned in #499, if you are interested in maintaining it as an extension, please let me know. Otherwise, if you migrate to v6, you can copy the code of the module and use it to your project.

**Pull request closed without merging**